packages/smart-contracts/scripts-create2/contract-setup/update-fee-proxies.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  singleRequestProxyFactoryArtifact,
+  erc20FeeProxyArtifact,
+  ethereumFeeProxyArtifact,
+} from '../../src/lib';
+import { HardhatRuntimeEnvironmentExtended } from '../types';
+import { getSignerAndGasFees } from './adminTasks';
+import { EvmChains } from '@requestnetwork/currency';
+import { executeContractMethod } from './execute-contract-method';
+import { Contract } from 'ethers';
+
+/**
+ * Update the proxy addresses in the SingleRequestProxyFactory contract
+ * @param hre Hardhat runtime environment
+ * @param signWithEoa Are transactions to be signed by an EOA
+ */
+export const updateFeeProxies = async (
```

Comment:
Please rename to `setupSingleRequestProxyFactory` to align with convention used by other setup scripts. Please rename the file too.